docs/static-rendering.md

```diff
@@ -0,0 +1,230 @@
+# Static rendering
+
+Client-side apps all suffer the same performance pitfall of not being able to show any content until the JavaScript is downloaded, parsed and run. The user sees a blank white screen until this has all occurred, which is often followed by a loading indicator while data is fetched from the network. To improve perceived performance, sku renders all the static content (everything not requiring an API call) of your app at build time.
+
+## Configuration
+
+The sku config contains route specific options.
+
+```js
+module.exports = {
+  routes: [
+    {
+      route: '/',
+      name: 'home',
+      entry: 'src/Home.js' // Optional: falls back to entry.client
+    },
+    {
+      route: '/details',
+      name: 'details',
+      entry: 'src/Details.js'
+    }
+  ],
+  environments: ['development', 'production'],
+  sites: ['au', 'jobstreet'],
```

Comment:
This may lead to confusion, having one code/locale and one brand. Maybe from a documentation perspective we go with `[ 'australia', 'asia' ]`